core/src/main/java/com/mapzen/android/graphics/OverlayManager.java

```diff
@@ -814,4 +871,13 @@ private MapData addPointToMarkerMapData(Marker marker) {
     findMe.setActivated(false);
     return false;
   }
+
+  @Override public boolean onRotate(float x, float y, float rotation) {
+    final View compass = mapView.findViewById(R.id.mz_compass);
```

Comment:
I created an issue for this and we can address it in a separate PR: https://github.com/mapzen/android/issues/242